igcommit/file_checks.py

```diff
@@ -248,3 +260,83 @@ def __str__(self):
         return '{} "{}" on {}'.format(
             type(self).__name__, self.args[0], self.committed_file
         )
+
+
+class FormatCheck(CommittedFileByExtensionCheck):
+    load_func = None
+    exception_cls = None
```

Comment:
Isn't it possible to have a default exception class? Something like `ValueError` like below or maybe a `CommitError`, `CheckError` or `FormatError`.